modules/virt-listing-vmis-cli.adoc

```diff
@@ -0,0 +1,17 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-listing-vmis-cli_{context}"]
+= Listing all virtual machine instances using the CLI
+
+You can list both virtual machine instances (VMIs) owned by virtual machines and standalone (VMIs) in your cluster by using the `oc` command-line interface (CLI).
+
+.Procedure
+
+. List all VMIs by running the following command:
```

Comment:
This is one step, so it should be a bullet point.

Reply:
HI @ousleyp - This particular topic is an existing topic inherited from the previous release of CNV. I don't recall making changes to this topic. Requesting clarification so I understand your feedback. 

Are you recommending that I take the sentence "List all VMIs.." and change it into a bullet?

Reply:
Yes, exactly. If it's one step, it needs to be a bullet point.

Reply:
Updated.